bench-tps/src/bench.rs

```diff
@@ -433,6 +454,14 @@ fn poll_blockhash<T: BenchTpsClient>(
         if blockhash_updated {
             let balance = client.get_balance(id).unwrap_or(0);
             metrics_submit_lamport_balance(balance);
+            datapoint_info!(
+                "poll_blockhash_stats",
```

Comment:
did you mean to put this one under the `blockhash_stats` datapoint like the other two?

Reply:
Yup, fixed.